docs/vault/installation.md

```diff
@@ -329,6 +326,33 @@ By default, the Vault will log at `info` or above but you may, for example, conf
 
 On startup, the Vault will automatically create or load the Bitcoin wallet using the keyname specified above and import additional keys generated from issue requests.
 
+#### Successful Startup
+
+When your start your Vault client, you should see logs similar to this:
+
+```sh
+Sep 19 11:05:59.799  INFO vault: Starting Prometheus exporter at http://127.0.0.1:9615
+Sep 19 11:05:59.799  INFO service: Version: 1.16.0
+Sep 19 11:05:59.799  INFO service: AccountId: a3d....
+Sep 19 11:05:59.799  INFO Server::run{addr=127.0.0.1:9615}: warp::server: listening on http://127.0.0.1:9615
+Sep 19 11:05:59.799  INFO bitcoin: Connecting to bitcoin-core...
+Sep 19 11:05:59.800  INFO bitcoin: Connected to main
+Sep 19 11:05:59.800  INFO bitcoin: Bitcoin version 220000
+Sep 19 11:05:59.807  INFO bitcoin: Waiting for bitcoin-core to sync...
+Sep 19 11:05:59.808  INFO bitcoin: Synced!
+Sep 19 11:05:59.808  INFO runtime::conn: Connecting to the btc-parachain...
+Sep 19 11:06:00.235  INFO runtime::conn: Connected!
+Sep 19 11:06:01.052  INFO runtime::rpc: spec_name=kintsugi-parachain
+Sep 19 11:06:01.052  INFO runtime::rpc: spec_version=1018000
+Sep 19 11:06:01.052  INFO runtime::rpc: transaction_version=3
+Sep 19 11:06:01.163  INFO runtime::rpc: Refreshing nonce: 1097
+Sep 19 11:06:01.496  INFO vault::system: Using 6 bitcoin confirmations
+Sep 19 11:06:01.496  INFO vault::system: Subscribing to error events...
+Sep 19 11:06:02.172  INFO vault::system: Adding derivation key...
+Sep 19 11:06:02.428  INFO vault::system: Adding keys from past issues...
+Sep 19 11:06:03.021  INFO vault::issue: Rescanning bitcoin chain from height 740163...
+```
```

Comment:
This is not the last step if the start-up initialization - we might want to add a couple more lines

Reply:
Agree - do you have the logs of a vault starting up at hand?